metabase/metabase
# Fixed various italian translations

### Description

This is a fix of a lot of erroneous italian translations. I have seen the poeditor advice only when i was opening the PR... :-(

`no-backport`

### How to verify

1. Open Metabase
2. Check italian labels

### Demo

![immagine](https://github.com/user-attachments/assets/1e9a6fcd-82ad-4817-9e83-e22976a9647a)

These are only a few examples...

### Checklist

None
